geoportailv3/static/js/authentication/authenticationdirective.js

```diff
@@ -64,76 +48,15 @@ app.AuthenticationController = function($http, loginUrl, logoutUrl,
     'login' : null,
     'password' : null
   };
```

Comment:
I'd recommend using two variables: `login` and `password`.

```js
/**
 * @type {string}
 * @export
 */
this.login = '';

/**
 * @type {string}
 * @export
 */
this.password = '';
```

Note the use of `@export` here. This is to be able to use the dot notation while preventing Closure Compiler from renaming the property.